cmds/root.go

```diff
@@ -26,7 +28,7 @@ func init() {
 var rootCmd = &cobra.Command{
 	Use:     "testtrack",
 	Short:   "TestTrack Split Config Management",
-	Long:    fmt.Sprintf("CLI for managing TestTrack experiments and feature gates\n\nVersion: %s\nBuild: %s", version, build),
+	Long:    fmt.Sprintf("CLI for managing TestTrack experiments and feature gates\n\nVersion: %s\nBuild: %s\nArch: %s", version, build, arch),
```

Comment:
this makes it so we can see what architecture the CLI was built for along with the specific version and build commit hash